mir_eval/melody.py

```diff
@@ -655,6 +725,14 @@ def evaluate(ref_time, ref_freq, est_time, est_freq, **kwargs):
         Time of each estimated frequency value
     est_freq : np.ndarray
         Array of estimated frequency values
+    est_voicing : np.ndarray
+        Estimate voicing confidence.
+        Default None, which means the voicing is inferred from est_freq:
+            frames with frequency <= 0.0 are considered "unvoiced"
+            frames with frequency > 0.0 are considered "voiced"
+    ref_reward : np.ndarray
+        Reference voicing reward.
```

Comment:
According to the metrics, wouldn't it be more accurate to say this is a pitch estimation reward, even if it is related to the (non-binary) voicing of the reference?